node/forcedroid.js

```diff
@@ -392,6 +394,50 @@ function createAppGradleFile(appFolderName, appName, usesSmartStore) {
 }
 
 //
+// Fixes library dependency references in the app's build.gradle files
+//
+function fixAppGradleFiles(appFolderName, appName, usesSmartStore) {
+    console.log('Tweaking build.gradle in ' + appFolderName + "/" + appName);
```

Comment:
All the ```Gradle``` dependencies are path based at this point, so tweaking is required post generation :-(